requirements.txt

```diff
@@ -1,4 +1,4 @@
-attr==0.3.1
+attrs==20.3.0
```

Comment:
not sure why this worked before and now I have to change it. But attrs is what we needed to pull in at this point, so this was changed